sdcm/nemesis.py

```diff
@@ -1082,13 +1074,28 @@ def add_ldap_configuration_to_node(node):
                 scylla_yaml.update(ldap_config)
             node.restart_scylla_server()
 
-        InfoEvent(message='Re-enable LDAP Authorization Configuration').publish()
-        create_ldap_container()
-        for node in self.cluster.nodes:
-            add_ldap_configuration_to_node(node)
+        try:
+            InfoEvent(message='Disable LDAP Authorization Configuration').publish()
+            for node in self.cluster.nodes:
+                remove_ldap_configuration_from_node(node)
+            destroy_ldap_container()
+
+            self.log.debug('Will wait few minutes with LDAP disabled, before re-enabling it')
+            time.sleep(600)
```

Comment:
i think this is not really necessary to move it into here, as the problem is usually adding the new container, rather than removing it.
but in any case, you could add in here in the `try` a call to this piece of code, as a separated function rather this?